themes/default/content/docs/support/troubleshooting.md

```diff
@@ -520,43 +362,140 @@ value defined elsewhere in your application. If the value is known, it can be co
 If the stack-reference-name truly is dynamic and cannot be known ahead of time to supply directly into the app, then this
 approach will not work, and the only way to workaround the issue is to follow the steps in [Use getOutput/requireOutput](#use-getoutput).
 
-## 403 HTTP error fetching plugin
+### Cannot connect to the Pulumi Service
```

Comment:
I think this section "Cannot connect to the Pulumi Service" should be moved up to follow (or maybe precede) the "Nothing happens due to network proxy" since they are both kind of networking related. It also puts the two topics related to pre-Pulumi 2.0 issues at the end.

Reply:
sure will do! was feeling a bit funny about this being at the bottom anyway.